### ISSUE TITLE
missing source distribution for 1.1.1 ?

### ISSUE THREAD

**Issue by igor47** [woodruffw/pyrage#47]
trying to install pyrage inside an alpine linux docker container, and i get:

```
2.710 ERROR: Could not find a version that satisfies the requirement pyrage==1.1.1 (from versions: 1.0.2rc2, 1.0.2)
2.711 ERROR: No matching distribution found for pyrage==1.1.1
```

this works fine if i use `pyrage==1.0.2` (by fine, i mean i also have to add `cargo` to the docker image, and then it takes a while to build a `musllinux` wheel).

any chance to add a source distribution for 1.1.1 ? even better would be a binary wheel for musllinux ;)

**Comment by woodruffw** [comment]
Hmm, looks like you're right: https://pypi.org/project/pyrage/#files

Sorry about that, I have no idea how that happened! I'll see about pushing a new point release with an sdist.

**Comment by woodruffw** [comment]
Ah, looks like the release workflow either needs to run `maturin sdist` or `maturin build --sdist`. SO it's a simple fix.

**Comment by woodruffw** [comment]
I've released 1.1.2, which includes an sdist: https://pypi.org/project/pyrage/1.1.2/#files

Thanks again for reporting this! 

(If you're interested, I'd be happy to accept a PR adding mullinux wheels -- but I have no idea how to do this myself.)
